scripts/import/import_EPMC.pl

```diff
@@ -736,6 +748,34 @@ sub clean_publications{
       }
     }
 
+    # Replace hexadecimal characters in title
+    if (defined $title_weird){
+      print $report "\nPublications with hexadecimal characters in the title - clean:\n";
+      foreach my $t_weird (@{$get_title_weird}){
+        my $pub_id = $t_weird->[0];
+        my $title = $t_weird->[1];
+        print $report "$pub_id\t$title\n";
+        my $new_title = replace_hex($title);
+
+        my $clean_title_sth = $dba->dbc()->prepare(qq[ update publication set title = ? where publication_id = $pub_id ]);
+        $clean_title_sth->execute($new_title);
```

Comment:
In case the title/authors still have those characters the script could write the warning in the report and skip the update instead of dying. These titles/authors can be cleaned after import and the import can be updated with new cases.

Reply:
Good idea